<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Estoque</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="buscaLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CÓDIGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="infoBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>481</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelCodigoText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CODIGO: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelQuantText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>QUANTIDADE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelValorText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>VALOR UNITÁRIO: R$</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCustoText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CUSTO: R$</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nomeButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ATUALIZAR NOME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="custoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ATUALIZAR CUSTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="incButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ADICIONAR QUANTIDADE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="decButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REMOVER QUANTIDADE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="precoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ATUALIZAR VALOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="codeText">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="quantText">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="valueText">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="costText">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelNameText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NOME:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameText">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelMarcaText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MARCA:</string>
     </property>
    </widget>
    <widget class="QLabel" name="marcaText">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="precoLineChange">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantIncLineChange">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>440</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantDecLineChange">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>480</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="costLineChange">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineChange">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>70</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="marcaButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ATUALIZAR MARCA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="marcaLineChange">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVER</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="productsTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>741</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>CÓDIGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QUANTIDADE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VALOR</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOME</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="refreshButtonTable">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineMarca">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="buscaLabel2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MARCA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
